apps/emqx_bridge_snowflake/src/emqx_bridge_snowflake_connector_schema.erl

```diff
@@ -0,0 +1,145 @@
+%%--------------------------------------------------------------------
+%% Copyright (c) 2024 EMQ Technologies Co., Ltd. All Rights Reserved.
+%%--------------------------------------------------------------------
+
+-module(emqx_bridge_snowflake_connector_schema).
+
+-behaviour(hocon_schema).
+-behaviour(emqx_connector_examples).
+
+-include_lib("typerefl/include/types.hrl").
+-include_lib("hocon/include/hoconsc.hrl").
+-include("emqx_bridge_snowflake.hrl").
+
+%% `hocon_schema' API
+-export([
+    namespace/0,
+    roots/0,
+    fields/1,
+    desc/1
+]).
+
+%% `emqx_connector_examples' API
+-export([
+    connector_examples/1
+]).
+
+%% API
+-export([]).
+
+%%------------------------------------------------------------------------------
+%% Type declarations
+%%------------------------------------------------------------------------------
+
+%%-------------------------------------------------------------------------------------------------
+%% `hocon_schema' API
+%%-------------------------------------------------------------------------------------------------
+
+namespace() ->
+    "connector_snowflake".
+
+roots() ->
+    [].
+
+fields(Field) when
+    Field == "get_connector";
+    Field == "put_connector";
+    Field == "post_connector"
+->
+    emqx_connector_schema:api_fields(Field, ?CONNECTOR_TYPE, fields(connector_config));
+fields("config_connector") ->
+    emqx_connector_schema:common_fields() ++ fields(connector_config);
+fields(connector_config) ->
+    Fields0 = emqx_connector_schema_lib:relational_db_fields(),
+    Fields1 = proplists:delete(database, Fields0),
+    Fields = lists:map(
+        fun
+            ({Field, Sc}) when Field =:= username; Field =:= password ->
+                Override = #{type => hocon_schema:field_schema(Sc, type), required => true},
+                {Field, hocon_schema:override(Sc, Override)};
+            ({Field, Sc}) ->
+                {Field, Sc}
+        end,
+        Fields1
+    ),
+    [
+        {server,
+            emqx_schema:servers_sc(
+                #{required => true, desc => ?DESC("server")},
+                ?SERVER_OPTS
+            )},
```

Comment:
Maybe we could infer this from the `account`, since there's no test container to be used.